python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -511,8 +492,44 @@ def resolve_tags(self, _graphene_info: ResolveInfo):
             if get_tag_type(key) != TagType.HIDDEN
         ]
 
-    def resolve_runStatus(self, _graphene_info: ResolveInfo) -> GrapheneRunStatus:
-        return GrapheneBulkActionStatus(self.status).to_dagster_run_status()
+    def resolve_runStatus(self, _graphene_info: ResolveInfo) -> str:
+        converted_status = BulkActionStatus[self.status]
+        if converted_status is BulkActionStatus.FAILED:
+            return GrapheneRunStatus.FAILURE
+        if converted_status is BulkActionStatus.CANCELED:
+            return GrapheneRunStatus.CANCELED
+        if converted_status is BulkActionStatus.CANCELING:
+            return GrapheneRunStatus.CANCELING
+        if converted_status is BulkActionStatus.REQUESTED:
+            # if no runs have been launched:
+            if len(self._get_records(_graphene_info)) == 0:
+                return GrapheneRunStatus.QUEUED
```

Comment:
should be not_started? starting? or just map to Started for all in requested state

Reply:
I think `STARTING` might make sense here, but if it's `STARTED`, it's not a big deal.